script.js

```diff
@@ -0,0 +1,86 @@
+const submit = document.getElementById('submit');
+const myBook = [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  addBook(newBook) {
+    this.myBook.push(newBook);
+  }
+
+  displayBook(newBook) {
+    const list = document.querySelector('#book-list');
+    const row = document.createElement('div');
+    row.innerHTML = `
+      <h4>${this.newBook.title}</h4>
+      <h4>${this.newBook.author}</h4>
+      <button><a href="#" class="btn btn-danger btn-sm remove">Remove</a></button><hr>`;
```

Comment:
Make sure that this button is working properly.

Reply:
Addressed this well!